samples/monster_generated.h

```diff
@@ -695,8 +697,22 @@ inline flatbuffers::TypeTable *MonsterTypeTable() {
     "equipped_type",
     "equipped"
   };
+  static const char* attr_keys_4[] = { "deprecated" };
+  static const char* attr_vals_4[] = { "0" };
+  static const flatbuffers::AttributeList field_attrs[] = {
+    {},
```

Comment:
This seems like a wasteful encoding if it is so sparse, since each unused item is 3 * sizeof(void *). Maybe better to make these pointers too, so unused ones can be nullptr